### ISSUE TITLE
Resolve nested `var` in fallback

### ISSUE THREAD

**Issue by asvny** [MadLittleMods/postcss-css-variables#37]
Just a spec clarification , are we allowed to refer fallback as another var ie `var(--color-2)`  from root

``` css
:root {
 --color-1:red;
 --color-2:green;
}

.Component {
  background: var(--color-10,var(--color-2))
}
```

I expect output to be 

``` css
.Component {
  background: green
}
```

but the current output is

``` css
.Component {
  background: var(--color-2)
}
```

**Comment by MadLittleMods** [comment]
Sorry for the slight delay. According to the spec, it looks like this should work but I can see that `postcss-css-variables` doesn't substitute it properly at the moment.

> if the var() function has a fallback value as its second argument, replace the var() function by the fallback value. If there are any var() references in the fallback, substitute them as well.
>
> https://drafts.csswg.org/css-variables/#using-variables

**Comment by annamalai-saro** [comment]
@MadLittleMods  I have made a pull request .. please review and then accept ... thank u !

**Comment by MadLittleMods** [comment]
Merged via https://github.com/MadLittleMods/postcss-css-variables/commit/d2bea891e34acd1bd2b655983db44186c3b0db55

**Comment by MadLittleMods** [comment]
Published in v0.7.0 https://github.com/MadLittleMods/postcss-css-variables/blob/master/CHANGELOG.md#v070---2017-3-12 (also on npm)

Thanks @asvny and @marklu for the contributions 😍
